e2e/tests/ui-driven/src/mocks/geospatialMocks.ts

```diff
@@ -37,3 +43,23 @@ export const mockMapGeoJson: Feature = {
     "organisation-entity": "13",
   },
 };
+
+export const mockChangedMapGeoJson: GeoJsonChangeHandler = {
```

Comment:
pulled this from the tests so I had a static mock to check against when running the tests